pages/page.py

```diff
@@ -33,11 +32,7 @@ def __init__(self, testsetup):
 
     @property
     def is_the_current_page(self):
-        if self._page_title:
-            page_title = self.page_title
-            Assert.equal(page_title, self._page_title,
-                         "Expected page title: %s. Actual page title: %s" %
-                         (self._page_title, page_title))
+        return self._page_title == self.page_title
```

Comment:
I was wondering the same. Not every page has `_page_title` defined, but those that don't we clearly don't call (or need to call) `is_the_current_page`. Otherwise, we would fail as the property would return `None`, which is falsey. If it did anything else it would be a false positive.